client/state/data-layer/wpcom/read/following/mine/test/index.js

```diff
@@ -151,14 +152,14 @@ describe( 'get follow subscriptions', () => {
 					blog_ID: 122463145,
 					URL: 'http://readerpostcards.wordpress.com',
 					feed_URL: 'http://readerpostcards.wordpress.com',
-					date_subscribed: Date.parse( '2017-01-12T03:55:45+00:00' ),
+					date_subscribed: moment( '2017-01-12T03:55:45+00:00' ),
```

Comment:
i'm not positive i love this. but Date.parse returns a timestamp (seconds since epoch), not a readable date. A moment instance serializes nicer and is easier to deal with in the debugger. Does mean we'll need custom serialization if/when we start persisting the follow set.